Sources/Swift/Protocol/SentryViewControllerBreadcrumbTracking.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+///
+/// Use this protocol to customize the name used in the automatic
+/// UIViewController performance tracker.
+///
+@objc
+public protocol SentryViewControllerBreadcrumbTracking: NSObjectProtocol {
+
+    /// The custom name of the UIViewController
+    /// that will be used for the transaction name.
+    var screenName: String { get }
```

Comment:
`m`: I think we should prefix this with Sentry to avoid name collisions.
```suggestion
    var sentryName: String { get }
```